### PR TITLE
Hts 1140 new reg

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosaveapi/validators/CreateAccountRequestValidator.scala [hunk 1]
```diff
@@ -65,7 +65,7 @@ class CreateAccountRequestValidator @Inject() (emailValidation: EmailValidation)
     }
 
     val bankDetailsCheck: ValidatedOrErrorString[Option[BankDetails]] =
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosaveapi/validators/CreateAccountRequestValidator.scala]: we don't need this check now - we want to accept bank details now

File: test/uk/gov/hmrc/helptosaveapi/services/HelpToSaveApiServiceSpec.scala [hunk 2]
```diff
@@ -120,12 +120,16 @@ class HelpToSaveApiServiceSpec extends TestSupport with MockPagerDuty {
   "The HelpToSaveApiService" when {
 
     val nino = "AE123456C"
-    val systemId = "MDTP"
+    val systemId = "MDTP-API-client"
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/helptosaveapi/services/HelpToSaveApiServiceSpec.scala]: where does `client` come from here?

File: test/uk/gov/hmrc/helptosaveapi/models/createaccount/CreateAccountBodySpec.scala [hunk 3]
```diff
@@ -39,7 +39,9 @@ class CreateAccountBodySpec extends TestSupport {
              |     "countryCode" : "country",
              |     "communicationPreference" : "preference"
              | },
-             | "registrationChannel" : "channel"
+             | "registrationChannel" : "channel",
+             | "version" : "2.0",
+             | "systemId" : "MDTP-API-ClientID"
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/helptosaveapi/models/createaccount/CreateAccountBodySpec.scala]: where does `ClientID` come from here?

File: app/uk/gov/hmrc/helptosaveapi/validators/CreateAccountRequestValidator.scala (hunk 1 above)
Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosaveapi/validators/CreateAccountRequestValidator.scala]: ok removed

File: test/uk/gov/hmrc/helptosaveapi/services/HelpToSaveApiServiceSpec.scala (hunk 2 above)
Comment (Jackie1988) [reply test/uk/gov/hmrc/helptosaveapi/services/HelpToSaveApiServiceSpec.scala]: this is to match whats in application.conf

File: test/uk/gov/hmrc/helptosaveapi/models/createaccount/CreateAccountBodySpec.scala (hunk 3 above)
Comment (Jackie1988) [reply test/uk/gov/hmrc/helptosaveapi/models/createaccount/CreateAccountBodySpec.scala]: its just test data, so its hard coded

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**